fhirstore/fhirstore.py

```diff
@@ -409,7 +409,7 @@ def comprehensive_search(self, resource_type: str, args: ImmutableMultiDict):
                             f"{e.info['error']['index']} is not indexed in the database yet."
                         )
 
-            bundle.append(included_hits, search_args.formatting_args)
+                        bundle.append(included_hits, search_args.formatting_args)
```

Comment:
I don't think this should be in the `except`

Reply:
true